Fix Trouble shooting guide on managed authentication users (#1787)
wso2/docs-choreo-dev: en/pe-docs/docs/references/troubleshoot-choreo.md

- ### After securing a web application with managed authentication,  I’m not able to add users who can sign in to the application.
   
      For step-by-step instructions on how to manage users with Choreo's built-in identity provider (IdP), see [Configure a User Store with the Built-In IdP](../api-management/consumer-idps/configure-a-user-store-with-built-in-idp.md).

>       If you have set up OpenID Connect (OIDC) supported identity providers (IdPs) for managed authentication, please navigate to the respective IdP's page to configure the users associated with it.

Reviewer: ```suggestion
      If you have set up OpenID Connect (OIDC) supported identity providers (IdPs) for managed authentication, refer to the respective IdP's documentation on how to configure the users associated with it.
```

Something like this? "Page" might not be a common term for every idp.
